README.md

```diff
@@ -223,5 +221,5 @@ guidelines][contributing].
 [pitest-maven]: https://pitest.org/quickstart/maven
 [pr-badge]: https://img.shields.io/badge/PRs-welcome-brightgreen.svg
 [refaster]: https://errorprone.info/docs/refaster
-[refaster-templates-bigdecimal]: error-prone-contrib/src/main/java/tech/picnic/errorprone/refastertemplates/BigDecimalTemplates.java
-[refaster-templates]: error-prone-contrib/src/main/java/tech/picnic/errorprone/refastertemplates/
+[refaster-templates-bigdecimal]: https://github.com/PicnicSupermarket/error-prone-support/blob/master/error-prone-contrib/src/main/java/tech/picnic/errorprone/refastertemplates/BigDecimalTemplates.java
```

Comment:
Absolute paths are required, as the same page is used to service our website.

Reply:
Clear! (I should probably just test this, but) will the build fail if we accidentally reintroduce a relative URL?

Reply:
Exactly why I introduced `html-proofer` s.t. we catch this stuff (didn't know the existence before, but came across in another repo and a very obvious check to introduce).

Reply:
Certainly! (And we shouldn't stop there. I made a note to also introduce `shellcheck`. We should likely enable other kind of static analysis.)